src/js/angular/import/directives/import-resource-tree.directive.js

```diff
@@ -180,6 +180,24 @@ function importResourceTreeDirective($timeout, ImportContextService) {
                 $scope.onEditResource({resource});
             };
 
+            $scope.toggleSelectAll = () => {
+                const allSelected = $scope.displayResources.every((resource) => resource.selected);
+                const noneSelected = $scope.displayResources.every((resource) => !resource.selected);
+                const checkboxElement = document.getElementById('checkboxInput');
```

Comment:
You shouldn't query the entire document. You have the directive's html element. Use it. But bare in mind that it's wrapped as a jquery object.